server/core/config/valid/global_cfg.go

```diff
@@ -17,10 +17,7 @@ const PoliciesPassedCommandReq = "policies_passed"
 const PlanRequirementsKey = "plan_requirements"
 const ApplyRequirementsKey = "apply_requirements"
 const ImportRequirementsKey = "import_requirements"
-const PreWorkflowHooksKey = "pre_workflow_hooks"
```

Comment:
why are you deleting these constants?

Reply:
Looked at the code base and it seems they are not being used.

Reply:
can you rever that, I believe those are being used.

Reply:
They are not being used. It's mainly used in the raw/global_cfg.go to parse the config. 

See: https://github.com/runatlantis/atlantis/blob/main/server/core/config/raw/global_cfg.go#L194-L195 as an example